client/styles/custom.scss

```diff
@@ -55,3 +55,7 @@ $tablet: (
   clear: both;
   height: 0;
 }
+
+.hide {
+  display: none !important;
```

Comment:
This was the only way I could get the USWDS conditional menus to work unfortunately -_- Displaying them using ternary statements broke the menu functionality.